normandy/recipes/urls.py

```diff
@@ -0,0 +1,14 @@
+from django.conf.urls import url, include
+
+from rest_framework.routers import DefaultRouter
+
+from normandy.recipes.api import ActionViewSet
+
+# API Router
+router = DefaultRouter()
+router.register(r'actions', ActionViewSet)
```

Comment:
Names of resources that occur in the URL should be singular. So this should be `/api/v1/action/` instead of `/api/v1/actions/`.